src/js/exercices.js

```diff
@@ -0,0 +1,202 @@
+let navElement = null;
+let selectedItem = null;
+let draggedElement = null;
+let rawAllOptions;
+let mooveX, mooveY;
+let id = 0;
+fetch('../../res/exerciceOptions.json')
+    .then((res) => res.json())
+    .then((json) => rawAllOptions = json);
+
+const draggables = document.getElementsByClassName("elements");
+const preview = document.getElementById('preview');
+const optionAside = document.getElementById('options');
+const jsonOutput = document.getElementById('jsonOutput');
+const title = document.getElementById('title-exo');
+const page = {
+    title: "Title Here",
+    elements: Array(),
+    height: "5cm",
+    idCategorie: "1"
+};
+
+
+title.addEventListener('input', (ev) => {
+    page.title = title.value;
+    jsonOutput.setAttribute('value', JSON.stringify(page));
+})
+document.getElementsByTagName('body')[0].addEventListener('keydown' , (ev) => {
+    if (ev.key === "Backspace" && selectedItem != null) {
```

Comment:
```suggestion
    if (ev.key === "Delete" && selectedItem != null) {
```
It's better with `supp` key